Completed homework 19 sprint)

RURURuina opened the pull request:
Если что-то неправильно подскажите, как решить)))
Заранее спасибо за проверку

RURURuina committed:
Completed homework 19 sprint)

Если что-то неправильно подскажите, как решить)))
Заранее спасибо за проверку

ArturNurtdinov commented on app/src/main/java/com/practicum/playlistmaker/search/ui/SearchFragment.kt:
```diff
@@ -284,4 +273,27 @@ class SearchActivity : AppCompatActivity() {
+    }
+
+
+    override fun onDestroy() {
```
В `onDestroyView` нужно очищать `_binding`, а не в `onDestroy` - https://developer.android.com/topic/libraries/view-binding#fragments

ArturNurtdinov commented on app/src/main/res/navigation/nav_graph.xml:
```diff
@@ -0,0 +1,27 @@
+        android:id="@+id/mainActivity"
+        android:name="com.practicum.playlistmaker.main.ui.MainActivity"
+        android:label="activity_main"
+        tools:layout="@layout/activity_main" />
```
Активити на самом деле нет в графе, активити держит весь граф, поэтому отсюда можно убрать эту активити

ArturNurtdinov reviewed (commented):
История поиска перестаёт отображаться после поиска, начинает отображаться только если переоткрыть экран:
https://github.com/RURURuina/PlaylistMaker/assets/49075521/c611e4e6-261e-4115-b716-259072248e0d
Ожидаемое поведения - после очищения поля ввода пользователь должен видеть актуальную историю поиска

RURURuina committed:
Completed homework 19 sprint)
После очистки история поиска появляется сразу (пофиксил)
Рекомендации тоже поправил
Если что-то неправильно подскажите, как решить)))
Заранее спасибо за проверку

RURURuina commented:
> История поиска перестаёт отображаться после поиска, начинает отображаться только если переоткрыть экран: https://github.com/RURURuina/PlaylistMaker/assets/49075521/c611e4e6-261e-4115-b716-259072248e0d Ожидаемое поведения - после очищения поля ввода пользователь должен видеть актуальную историю поиска

Спасибо, что нашли
Не заметил этот баг
